driver-core/src/main/com/mongodb/internal/operation/ServerVersionHelper.java

```diff
@@ -25,34 +25,18 @@
 public final class ServerVersionHelper {
 
     public static final int MIN_WIRE_VERSION = 0;
```

Comment:
The naming of this is weird or should it mirror `FOUR_DOT_ZERO_WIRE_VERSION` similar to `LATEST_WIRE_VERSION` does.